recipes/quickbin/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "quickbin" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/quickbin-{{ version }}.tar.gz
+  sha256: 21246a61b653490e69ebef712978d7167d5ec9412f4d414c3da325750433dd3f
+
+build:
+  skip: true  # [py<311 or win or linux32]
```

Comment:
```suggestion
  skip: true  # [py<311 or win]
```

Reply:
I don't have 32-bit wheels built for this. Will the build pipeline automatically detect this?

Reply:
trying it out.